topics/icrc_34_delegation.md

```diff
@@ -1,50 +1,60 @@
-  * [Request](#request)
-    * [Example RPC Request](#example-rpc-request)
-    * [Response](#response)
-    * [Example RPC Response](#example-rpc-response)
+    * [Example RPC Permission Request](#example-rpc-permission-request)
+  * [icrc-25-supported-standards](#icrc-25-supported-standards)
+  * [Request params](#request-params)
+    * [Example RPC Delegation Request](#example-rpc-delegation-request)
+  * [Result](#result)
+    * [Example RPC Delegation Response](#example-rpc-delegation-response)
   * [Message Processing](#message-processing)
   * [Errors](#errors)
 <!-- TOC -->
 
 ## Summary
 
-When a relying party wants to authenticate as a user, it uses
-a [session key](https://internetcomputer.org/docs/current/references/ic-interface-spec/#signatures), and
-below `icrc34_get_global_delegation` method to obtain a delegation chain that allows the session key to sign for the
-user's global identity. The obtained delegation chain **MUST** be restricted by the signer to canister targets that
-trust the relying party following the [ICRC-28](./icrc_28_trusted_origins.md) standard.
+When a relying party wants to avoid going through the signer and user approval for each individual call, it can use the icrc34_delegation method described below. 
 
-- Due to the canister target restriction, only the canisters that the relying party owns can be called on behalf of the user.
-- A new delegation chain must be obtained for every new canister created by the relying party.
-- The identity is the same across all relying parties, the user is not anonymous between different relying parties.
+This method generates a delegation chain, an object with a stable identifier, allowing the relying party to sign transactions on behalf of the user without having to go through the signer and user approval. This also means the signer does not need to stay connected to the signer to make calls on behalf of the user.
 
-## Method
+The delegation chain's stable identifier can be either: 
+1. **Account Delegation**: an identity that may hold tokens and other assets.
+2. **Relying Party Delegation**: an identity designed exclusively for the relying party (Relying Party Delegation).
 
-**Name and Scope:** `icrc34_get_global_delegation`
+Example use cases of a Relying Party Delegation:
+- Signer could give the user a choice to stay anonymous with a service.
+- Differentiate between calls made with user approval (Account identifier) and without user approval (Relying Party Delegation identifier). Allowing for fine grained security levels per identifier on the Relying Party platform.
+- Exclusive identifier within the Relying Party platform to stay isolated from identifiers of other Relying Party platforms.
 
-**Prerequisite:** Active session with granted permission scope `icrc34_get_global_delegation` or `*`.
+If a relying party wants to receive an Account Delegation, the `icrc34_delegation` request **MUST** include canisters it
```

Comment:
The "it controls" part is not a requirement per ICRC-28, from the ICRC-28 perspective the developer must simply make sure it is not malicious.
If you want to explicitly restrict ICRC-28 to only be used by canisters under the exact same control structure, then that should be reflected there as well.

Reply:
updated